DEFI/ERC4626/README.md

```diff
@@ -0,0 +1,12 @@
+# ERC4626 - Full Spec
+
+A specification file of ERC4626.
+
+This spec can be checked against implementations of ERC4626.
+
+We have an implementation of an ERC4626 contract in `certora/harnesses/mixins/ERC4626AccountingHarness.sol`.
```

Comment:
we wanted also to run on the openzeppline code
 https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/extensions/ERC4626.sol